db/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
An alternative to this approach would be: Add the covers to the Books entity from a CDS file within the srv directory. I usually would prefer to use the plugins where defined and avoid defining plugin dependencies in root pom.xml. If we discuss that this is not a good pattern (e.g. adding Attachments from a source file within srv directory) we should rather consider changing or making configurable the behaviour of visibility of plugin CDS files (e.g. visible to db folder by default/by configuration in the resolve goal)